pkg/tsdb/grafana-testdata-datasource/scenarios.go

```diff
@@ -902,7 +903,7 @@ func predictableSeries(timeRange backend.TimeRange, timeStep, length int64, getV
 	wavePeriod := timeStep * length
 	maxPoints := 10000 // Don't return too many points
 
-	timeVec := make([]*time.Time, 0)
+	timeVec := make([]time.Time, 0)
```

Comment:
Should this scenario also have the frame meta type set?

Reply:
The `predictableSeries` (and also the `predictableCSVWave`) aren't causing any problems for SQL Expressions right now. So personally I'd like to leave changing those outside the scope of this PR.

But yes, I think **everything** should have the frame meta type set, in an ideal world. But planning to leave that up to data source owners to take care of 🙏

Reply:
> The predictableSeries (and also the predictableCSVWave) aren't causing any problems for SQL Expressions right now. So personally I'd like to leave changing those outside the scope of this PR.

That is not quite true, once you add a label to either of them (or add a second series in csvwave), you will trigger the error -- so would be good to add the type here as well.

Reply:
Also note then once the type is added, you will still see different conversion behavior based on adding a second frame or labels until we address https://github.com/grafana/grafana/issues/103124